contributed_definitions/nyaml/NXactuator.yaml

```diff
@@ -0,0 +1,198 @@
+category: base
+doc: |
+  An actuator used to control an external condition.
+  
+  The condition itself is described in :ref:`NXenvironment`.
+type: group
+NXactuator(NXobject):
+  model:
+    doc: |
+      Actuator identification code/model number
+  name:
+    doc: |
+      Name of the actuator
+  short_name:
```

Comment:
Went with `short_name` here to have the same structure as `NXsensor` (which we don't want to modify since it is part of the NIAC standard already for quite some time).

Reply:
a short_name of something is an alias, indeed maybe that comment would have been better placed in a change request to NXsensor and NXdetector with similar points like vendor instead of using NXfabrication, it is a related but different issue